compiler/src/iree/compiler/Codegen/Common/TransformExtensions/CommonExtensions.cpp

```diff
@@ -1360,5 +1360,174 @@ void transform_dialect::IREEEraseHALDescriptorTypeFromMemRefOp::getEffects(
   transform::modifiesPayload(effects);
 }
 
+//===---------------------------------------------------------------------===//
+// ConvertConv2DToImg2ColAndAdjustWorkgroupCountOp
+//===---------------------------------------------------------------------===//
+
+// Rewrite the workgroup count compute region based on the specified collapsed
+// dimensions on the output tensor. Only the dims that are expected to be tiled
+// and distributed are adjusted, thus considering only the output is sufficient.
+static LogicalResult adjustWorkgroupCountComputeRegionForImg2Col(
```

Comment:
Could you please add a TODO to say this is to be deprecated after workgroup count is computed using slices....